html/feed/feed.js

```diff
@@ -24,3 +24,60 @@ async function getPosts (headerOptions){
 }
 
 getPosts(options);
+
+
+
+/* this is the js from hell part 2 */
+
+/*---- structure from html for th card  --- transform this into js*/
+
+
+async function createCards (){
+        const posts = await getPosts()
```

Comment:
You are not seing anything because you call the function with no options, you call the function outside the createcards function with the correct parameters. You dont need to call the function two times. You want the data to use inside the createCards function so you only need to call it inside that function, and with the correct parameters, in this case options.

Reply:
this code is removed